Tools/CISupport/ews-build/steps.py

```diff
@@ -3781,6 +3785,14 @@ def run(self):
 
     def evaluateCommand(self, cmd):
         rc = super().evaluateCommand(cmd)
+
+        is_main = self.getProperty('github.base.ref', DEFAULT_BRANCH) == DEFAULT_BRANCH
```

Comment:
Let's move this block of code inside `run()` (just before `defer.returnValue(rc)`)